### ISSUE TITLE
dedicated server crash: attempt to index local 'ws' (a nil value)

### ISSUE THREAD

**Issue by credomane** [kryojenik/LTN_Combinator_Modernized#42]
Looking on a dedicated mp server and we keep getting the below crash. Originally we were on ltn combinator 2.0.3 but updated after the first crash.

I can't replicate the bug nor can the original player when in single player. But playing on the mp server the other player whenever they set a ltn combinator to request beryl ore (from space exploration) it crashes. The single player testing was even done with the exact same the server is using. We are stumped.

```
`Error while running event LTN_Combinator_Modernized::on_gui_click (ID 1)
__LTN_Combinator_Modernized__/script/ltn_combinator.lua:600: attempt to index local 'ws' (a nil value)
stack traceback:
    __LTN_Combinator_Modernized__/script/ltn_combinator.lua:600: in function 'handler'
    __LTN_Combinator_Modernized__/script/ltn_combinator.lua:833: in function 'wrapper'
    __flib__/gui-lite.lua:139: in function 'handler'
    __flib__/gui-lite.lua:169: in function 'handler'
    __core__/lualib/event_handler.lua:47: in function <__core__/lualib/event_handler.lua:45>`
```

Sorry if anything is formatted weird. Trying do this from my phone while at a rest stop.

**Comment by credomane** [comment]
We ended up using a constant combinator to work around the issue for now.

**Comment by kryojenik** [comment]
@credomane Any chance you can provide the world file that this happens with, and perhaps a location / which combinator if a specific one?  If your user is able to reproduce it, maybe it would help to watch them do it while streaming?

**Comment by credomane** [comment]
I saw your reply while I was outta town doing Mother's Day stuff and by the time I got home hours later I forgot all about it.

Here's the file in question [CommunityMap-2016July.zip](https://drive.google.com/file/d/0B7gnuvnN_v5SSW5xcVpDX0FUbFk/view?resourcekey=0-P7RbwVRsRKPxAkVKl4tYXw) It is too large for github so it is hosted on google drive.

The train stop in question is at `[gps=-9.6,-212.1,Rokenharr]` but it has been converted to a normal constant combinator. I don't have a save from which is was still a LTN combinator.

**Comment by kryojenik** [comment]
This is likely fixed by 20871ab4.  At least mitigated to only logging an error and not crashing.  If this error does continue to show, please let me know, and what was going on at the time.

The specific steps to reproduce #43 which was very similar:

Click on a signal button to create a request (do not click confirm or type enter)
Click on a Provider / Requester / Depot check box
Finally, click on the confirn button that is still enabled (but now no request signal is being set up)
Crash 

Those steps  as of 2.0.7 will not be possible because the confirm button is not left enabled.   Anything along those lines will be helpful to track this down if it is something different.

**Comment by kryojenik** [comment]
Closing per previous comment
